keylime/ima/ima_dm.py

```diff
@@ -367,6 +366,22 @@ def invalid(self) -> Failure:
         return failure
 
 
+def _strtobool(val):
+    """Convert a string representation of truth to true (1) or false (0).
+
+    True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
+    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
+    'val' is anything else.
+    """
+    val = val.lower()
+    if val in ("y", "yes", "t", "true", "on", "1"):
+        return 1
```

Comment:
Should return True?

Reply:
This is the same function as in the python standard lib, but I changed that because it does not make a difference is our case.